aiohttp/web_protocol.py

```diff
@@ -65,7 +65,16 @@
 ]
 
 ERROR = RawRequestMessage(
-    "UNKNOWN", "/", HttpVersion10, {}, {}, True, False, False, False, yarl.URL("/")
+    "UNKNOWN",
+    "/",
+    HttpVersion10,
+    {},  # type: ignore
+    {},  # type: ignore
```

Comment:
Could you specify which errors are getting ignored? I saw syntax like `# type: ignore:SomeError` somewhere but I can't recall if it's documented.

Reply:
Done. I was not aware of this syntax.